community/samples/serving/helloworld-elixir/README.md

```diff
@@ -79,8 +79,7 @@ When asked, if you want to `Fetch and install dependencies? [Yn]` select `y`
    # Run everything else as 'appuser'
    USER appuser
 
-   # Document that the service listens on port 8080.
-   ENV PORT=8080 MIX_ENV=prod REPLACE_OS_VARS=true
+   MIX_ENV=prod REPLACE_OS_VARS=true
```

Comment:
same here

Reply:
added the key at both the places. Thanks for pointing out!